Packs/SandBlastAppliance/pack_metadata.json

```
@@ -18,5 +18,6 @@
     "marketplaces": [
         "xsoar",
         "marketplacev2"
-    ]
+    ],
+    "hidden": true
```

Do not hide the pack yet.
We are allowing customers to migrate before removing the pack from the marketplace.
```suggestion
    ]
```